src/handlers/PreviewerHandler.ts

```diff
@@ -115,14 +116,17 @@ export class PreviewerHandler {
             this.modify,
             this.threadId
         );
-
-        const items = res.map((item) => {
-            return {
-                id: item.prompt,
-                type: SlashCommandPreviewItemType.TEXT,
-                value: item.prompt,
-            };
-        });
+        let items: ISlashCommandPreviewItem[] = [];
+
+        if (res) {
+            items = res.map((item) => {
+                return {
+                    id: item.prompt,
+                    type: SlashCommandPreviewItemType.TEXT,
+                    value: item.prompt,
+                };
+            });
+        }
```

Comment:
@AmanNegi What if the res is not there how user will get to know ? as discussed.